crates/forge/tests/data/contracts/trace_info_proxy.cairo

```diff
@@ -0,0 +1,48 @@
+use starknet::{ContractAddress, ClassHash};
+
+#[starknet::interface]
+trait ITraceInfoProxy<T> {
+    fn with_libcall(ref self: T, class_hash: ClassHash) -> felt252;
```

Comment:
Does this need to be ref?

Reply:
No, but it doesn't make any difference I suppose